src/ui/bar-chart/BarChart.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactNode, useEffect, useState, Children } from 'react';
+
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+
+import csx from "./BarChart.scss"
+
+interface IBarChart {
```

Comment:
We don't use IInterface convention

Reply:
I forgot replace it with namespace and Props name